data/data_generator.rb

```diff
@@ -0,0 +1,46 @@
+require 'ffaker'
+require 'date'
+require_relative '../dependency'
+
+module DataGenerator
+  def generate_authors(quantity = 10)
+    authors = []
+
+    quantity.times { authors << Author.new(FFaker::Name.name, 'Biography') }
```

Comment:
You can use just `quantity.times.map { #do_something }` instead of pulling object to empty array